packages/shared/src/utils/routes/routes.ts

```diff
@@ -78,13 +77,14 @@ export const routes = {
     compare_cfds: '/cfd-compare-acccounts',
 
     // Wallets
-    wallets: '/wallets',
-    wallets_cashier: '/wallets/cashier',
-    wallets_deposit: '/wallets/cashier/deposit',
-    wallets_withdrawal: '/wallets/cashier/withdraw',
-    wallets_transfer: 'wallets/cashier/transfer',
-    wallets_transactions: '/wallets/cashier/transactions',
-    wallets_compare_accounts: '/wallets/compare-accounts',
+    wallets: '/wallet',
+    wallets_deposit: '/wallet/deposit',
+    wallets_withdrawal: '/wallet/withdrawal',
+    wallets_transfer: 'wallet/account-transfer',
```

Comment:
https://github.com/binary-com/deriv-app/pull/15204
```suggestion
    wallets_transfer: '/wallet/account-transfer',
```